src/styles.scss

```diff
@@ -239,5 +239,25 @@ body {
     height: 50px;
     padding: 3px;
   }
+  
+  .space-container {
+    margin: 64px 30px;
+  }
+
+  .mat-column-select {
+    max-width: 44px;
+  }
 
+  .mat-column-action, .mat-column-rating{
+    max-width: 225px;
+  }
+  
+  .tableAnchor a:hover {

```

Comment:
We should use a more meaningful class name and kebab-case rather than camelCase for consistency.  Maybe `.primary-link-hover` unless I think of something better tonight